octoprint_mrbeam/static/js/helpers/working_area_helper.js

```diff
@@ -1,4 +1,48 @@
 class WorkingAreaHelper {
+    static versionCompare(v1, v2, options) {
+        var lexicographical = options && options.lexicographical,
```

Comment:
it would be nice to rename this to areLexicographicalVersions to know we are expecting to find a boolean here

Reply:
we actually changed the whole handling. Will send you a new review request once done.